index.html

```diff
@@ -12,54 +12,71 @@
   <div class="landing-page">
     <header>
       <nav class="nav">
-        <div class="nav__logo">my logo</div>
-        <div class="nav__menu-box">
-          <div class="nav__menu nav__menu--line-1"></div>
-          <div class="nav__menu nav__menu--line-2"></div>
-          <div class="nav__menu nav__menu--line-3"></div>
+        <div class="nav__logo"><a href="#" rel="noopener noreferrer">my logo</a></div>
+        <div class="nav__menu-line">
+          <div class="nav__menu"></div>
+          <div class="nav__menu"></div>
+          <div class="nav__menu"></div>
```

Comment:
[OPTIONAL] For some reason the menu list does not change to the `hamburger` button on mobile view. Kindly consider looking into it to improve your project. You can use `display: none;` to hide the menu list on mobile view and vice versa.

![image](https://user-images.githubusercontent.com/38088818/165978638-69f45662-f136-4976-8521-29fa4ead7da9.png)

Otherwise, fantastic job 👏🏾